docs/lib/datastore/fragments/js/other-methods.md

```diff
@@ -0,0 +1,19 @@
+## Clear
+
+To clear local data from DataStore, use the `clear` method:
+
+```js
+import { DataStore } from '@aws-amplify/datastore';
+
+await DataStore.clear();
+```
+
+## Start
+
+To manually start the sync process, use the `start` method:
```

Comment:
We might want something like "_Synchronization starts automatically whenever you run `DataStore.query()` however you can explicitly begin the process with `DataStore.start()`_"

@manueliglesias can you confirm this behavior is still the case?

Reply:
Yes, that's right.

Reply:
cc @manueliglesias I'm adding this line into the PR, if this is no longer the case please let me know so I can update or remove it from the docs:

```md
Synchronization starts automatically whenever you run `DataStore.query()` however you can explicitly begin the process with `DataStore.start()`.
```

Reply:
@dabit3 Sounds good! At some point it would be good to show how to wait for the sync process to finish (using Hub) because if your first-ever operation is a query, you might get back an empty result if the sync hasn't fetched enough data t that point in time.